### PR TITLE
Very cool and good PR

### PR DESCRIPTION
**Pull request by jpmorgan98**
# Some very important changes:

- Added specific folder to collect elements
- Added a diagram demonstrating the use case
- Adding specific file for Uranium and its *very real isotopes*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+| |_| | |\ \ /\__/ /\__/ /_| |_  \  /\  /_| |_| |\  | | | | |___| |\ \ 
+ \___/\_| \_|\____/\____/ \___/   \/  \/ \___/\_| \_/ \_/ \____/\_| \_|
+                                                                       
+                                                                       
+ _____ _____  _   _ _____  _____ _                                     
+/  ___/  __ \| | | |  _  ||  _  | |                                    
+\ `--.| /  \/| |_| | | | || | | | |                                    
+ `--. \ |    |  _  | | | || | | | |                                    
+/\__/ / \__/\| | | \ \_/ /\ \_/ / |____                                
+\____/ \____/\_| |_/\___/  \___/\_____/
```
Comment (munkm) [inline README.md:16]: ASCII art is awesome!!! This is such a great improvement to the README

File: README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ _____ _____  _   _ _____  _____ _                                     
+/  ___/  __ \| | | |  _  ||  _  | |                                    
+\ `--.| /  \/| |_| | | | || | | | |                                    
+ `--. \ |    |  _  | | | || | | | |                                    
+/\__/ / \__/\| | | \ \_/ /\ \_/ / |____                                
+\____/ \____/\_| |_/\___/  \___/\_____/                                
+
+```
+
+Whoop whoop whoes accited to do computer
```
Comment (munkm) [inline README.md]: Oh! I think there's a typo here. I think this fixes it. 

```suggestion
Whoop whoop who's excited to do computer??!!?!??
```

File: elements/elements.md [hunk 3]
```diff
@@ -3,7 +3,6 @@ helium
 carbon is a metal? 
 thorium 
 protactinium
```
Comment (munkm) [inline elements/elements.md:5]: I like that you added a folder for the elements file. But Uranium is still an element. Could you please add it back in to the file?

File: elements/uranium_isotopes.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+uranium-1
+uranium-tab
+uranium-hmmmmmmm
+uranium2cool4winter_skool
```
Comment (munkm) [inline elements/uranium_isotopes.md]: Hmmm. I don't know these U isotopes. I've suggested the ones I know below. Would you mind clarifying what you meant here? 

```suggestion
uranium-233
uranium-235
uranium-238
uranium-239
```

**Review by munkm — CHANGES_REQUESTED** [review]
I really like your enhancements here!!!! Thank you so much for the ascii art. I noticed a few typos and I had a question about the U isotopes.

File: README.md (hunk 2 above)
Comment (jpmorgan98) [reply README.md]: Ya those look great!

File: elements/uranium_isotopes.md (hunk 4 above)
Comment (jpmorgan98) [reply elements/uranium_isotopes.md]: While I can add those in I needed those uranium the previous ones should also remain for other data handling functionality [see here](google.com)

File: elements/elements.md (hunk 3 above)
Comment (jpmorgan98) [reply elements/elements.md:5]: got it but I do think it should be at the bottom cus its so heavy

**Review by jpmorgan98 — COMMENTED** [review]
I mostly agree with your changes but I do need those seemingly odd uranium isotopes for data handle. I have added some documentation

**Review by munkm — APPROVED** [review]
This looks great!

**Pull request merged**